### PR TITLE
Mp telemetry1.0

### PR DESCRIPTION
**Pull request by Yas-Franco**
For issue #19172

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#build

File: dev/io.openliberty.com.squareup.okhttp/bnd.bnd [hunk 1]
```diff
@@ -10,32 +10,26 @@
...
-  !zipkin2.*,\
-  !kotlin,\
-  !kotlin.*,\
-  okhttp3,\
+  !android.net.http,\
+  javax.net,\
   javax.net.ssl,\
-  *
+  kotlin.*,\
+  okio
```
Comment (tevans78) [inline dev/io.openliberty.com.squareup.okhttp/bnd.bnd:24]: What happens to this bundle if you just leave the imports as `*`?

File: dev/io.openliberty.com.squareup.okio-jvm/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Bundle-Name: Square Okio JVM
+Bundle-SymbolicName: io.openliberty.com.squareup.okio-jvm
+Bundle-Description: Liberty, Square Okio JVM version ${bVersion}; Jakarta Enabled
+
+bVersion=1.0
+
+Import-Package: \
+  !android.net.http,\
+  javax.net,javax.net.ssl,\
+  kotlin.*
```
Comment (tevans78) [inline dev/io.openliberty.com.squareup.okio-jvm/bnd.bnd]: Can this be `*`?

File: dev/io.openliberty.io.opentelemetry.internal/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+-include ~../cnf/resources/bnd/transform.props
+bVersion=1.0
+openTelemetryVersion=1.14.0
+
+Bundle-SymbolicName: io.openliberty.io.opentelemetry.internal; singleton:=true
+
+Import-Package: \
+  !io.opentelemetry.exporter.prometheus,\
+  javax.net,javax.net.ssl, \
```
Comment (tevans78) [inline dev/io.openliberty.io.opentelemetry.internal/bnd.bnd]:
```suggestion
  javax.net, \
  javax.net.ssl, \
```

File: dev/io.openliberty.io.opentelemetry.internal/bnd.bnd [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+#*******************************************************************************
+
+-include ~../cnf/resources/bnd/transform.props
+bVersion=1.0
+openTelemetryVersion=1.14.0
+
+Bundle-SymbolicName: io.openliberty.io.opentelemetry.internal; singleton:=true
+
+Import-Package: \
+  !io.opentelemetry.exporter.prometheus,\
```
Comment (tevans78) [inline dev/io.openliberty.io.opentelemetry.internal/bnd.bnd]: I don't think you need to exclude anything if you're not using a `*` in the imports

File: dev/io.openliberty.org.jetbrains.annotation/.project [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>io.openliberty.io.grpc.netty.shaded</name>
```
Comment (tevans78) [inline dev/io.openliberty.org.jetbrains.annotation/.project]:
```suggestion
	<name>io.openliberty.org.jetbrains.annotation</name>
```

File: dev/io.openliberty.org.jetbrains.annotation/bnd.bnd [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+Bundle-Name: JetBrains Annotation Jakarta
+Bundle-SymbolicName: io.openliberty.org.jetbrains.annotation
+Bundle-Description: Liberty JetBrains Annotation, version ${bVersion}; Jakarta Enabled
+
+bVersion=1.0
+
+Import-Package: \
+  !android.net.http,\
+  javax.net,\
+  javax.net.ssl
```
Comment (tevans78) [inline dev/io.openliberty.org.jetbrains.annotation/bnd.bnd]: Either use `*` or remove the excluded package. Are you sure that the `javax` packages are needed here.

File: dev/io.openliberty.org.jetbrains.kotlin/.project [hunk 7]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>io.openliberty.io.grpc.netty.shaded</name>
```
Comment (tevans78) [inline dev/io.openliberty.org.jetbrains.kotlin/.project]:
```suggestion
	<name>io.openliberty.org.jetbrains.kotlin</name>
```

File: dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+Bundle-Name: JetBrains Kotlin Jakarta
+Bundle-SymbolicName: io.openliberty.org.jetbrains.kotlin
+Bundle-Description: Liberty JetBrains Kotlin, version ${bVersion}; Jakarta Enabled
+
+bVersion=1.0
+
+Import-Package: \
+  !android.net.http,\
+  javax.net,\
+  javax.net.ssl
```
Comment (tevans78) [inline dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd]: Same questions as above

File: dev/io.openliberty.com.squareup.okio-jvm/.project [hunk 9]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>io.openliberty.microprofile.telemetry.internal.common</name>
+	<name>io.openliberty.io.grpc.netty.shaded</name>
```
Comment (tevans78) [inline dev/io.openliberty.com.squareup.okio-jvm/.project]:
```suggestion
	<name>io.openliberty.com.squareup.okio-jvm</name>
```

**Comment by tevans78** [comment]
You need to check all your .project files, including this one - dev/io.openliberty.com.squareup.okhttp/.project - they have the wrong actual project name in

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
#build

File: dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+Bundle-Description: Liberty JetBrains Kotlin, version ${bVersion}; Jakarta Enabled
+
+bVersion=1.0
+
+Import-Package: \
+  *
+
+Export-Package: \
+  kotlin;version="1.0.0",\
+  kotlin.*;version="1.0.0"
```
Comment (tevans78) [inline dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd]: Versions are still wrong but you can fix that in a follow up PR
```suggestion
  kotlin;version="1.6.20",\
  kotlin.*;version="1.6.20"
```

File: dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+instrument.disabled: true
+
+-fixupmessages.missingexport: "Used bundle version * for exported package";is:=ignore
+
+-buildpath: \
+  org.jetbrains.kotlin:kotlin-stdlib;version=1.6.20,\
+  org.jetbrains.kotlin:kotlin-stdlib-common;version=1.6.2
```
Comment (tevans78) [inline dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd]: This should have been 1.6.20

File: dev/cnf/oss_dependencies.maven [hunk 12]
```diff
@@ -620,6 +619,9 @@ org.jboss.weld:weld-osgi-bundle:2.4.8.Final
 org.jboss.weld:weld-osgi-bundle:3.1.9.Final
 org.jboss.weld:weld-osgi-bundle:4.0.3.Final
 org.jboss.weld:weld-osgi-bundle:5.0.1.Final
+org.jetbrains.kotlin:kotlin-stdlib-common:1.6.2
```
Comment (tevans78) [inline dev/cnf/oss_dependencies.maven]: Should have been 1.6.20

File: dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd (hunk 10 above)
Comment (tevans78) [reply dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd]: Need to fix the version of `org.jetbrains.kotlin:kotlin-stdlib-common;version=1.6.2` so might as well fix this too

**Comment by Yas-Franco** [comment]
#build

**Review by tevans78 — APPROVED** [review]

**Comment by tevans78** [comment]
#libby

**Pull request merged**